### ISSUE TITLE
CLI Search Fails Silently on Enter

### ISSUE THREAD

**Issue by espii-ainen** [layday/instawow#131]
When using the instawow CLI, the addon search and install navigation is somewhat unintuitive.

If you select an addon from the offered choices with arrow keys, it would be fairly intuitive to press enter to select/install the choice, however that just makes the tool fail silently without explaining what happened.

I believe the tool should exit notifying the user to mark addons to install with the space key.

```
╰─ instawow search trp3
Select add-ons to install  (use arrow keys to move, <space> to select, <o> to view in your browser)
   ○ tRP3_RDotIO  (github:ninerbull/trp3_rdotio==v1.4.6)
 » ○ Total RP 3  (wowi:24113-total-rp-3==2.5.2)
   ○ Total-RP-3  (github:total-rp/total-rp-3==2.5.2)
```

**Comment by layday** [comment]
The reason it doesn't work with enter is that you can select multiple add-ons to install.  I can display a message on exit; thanks for the suggestion.

**Comment by espii-ainen** [comment]
Oh yeah, I totally get the reasoning now! It just stumped me for a moment before I realized how it works. I figured other people might be confused what the intended usage was as well so I wanted to at least bring it up!

**Comment by layday** [comment]
Addressed in v3.0.0.
